BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -736,7 +745,6 @@ def main():
     cmd_display.add_argument('-c',  dest='compress', choices=['lz4', 'lzma', 'dummy'],  default='dummy', help='compression algorithm')
     cmd_display.add_argument('-a',  dest='auth', choices=['SHA2_256', 'SHA2_384', 'RSA2048', 'RSA3072' 'none'], default='none',  help='authentication algorithm')

```

Comment:
Let's be more specific in the help.

Update this line to this:
cmd_display.add_argument('-a',  dest='auth', choices=['SHA2_256', 'SHA2_384', 'RSA2048_SHA2_256', 'RSA3072_SHA2_384' 'none'], default='none',  help='authentication algorithm')